boards/arm/adafruit_feather_m0_basic_proto/adafruit_feather_m0_basic_proto.yaml

```diff
@@ -3,7 +3,7 @@ name: Adafruit Feather M0 Basic Proto
 type: mcu
 arch: arm
 ram: 32
-flash: 256
+flash: 232
```

Comment:
How come you are reducing the amount of flash reported?

Reply:
Not finding the comment (deleted?), but someone had mentioned that the size of the code partition had to be taken into account to prevent some tests from failing. Code partition is declared in the device tree with a size of 0x3A000 = 232kB. This is where the value comes from.